### DISCUSSION TITLE
Downloaded theme is not really what  I see

### DISCUSSION THREAD

**Discussion by Harry-nl** [rejetto/hfs#632] [Q&A]
Hoi all, Because I'm totally new to this system, I have the server working without problems .
Now I have downloaded the dragon theme plugin and activated it. Its change the colors but not the completly style, see the images what I mean. 
![screen_845](https://github.com/rejetto/hfs/assets/172913982/7b6b3712-0b50-4411-9e98-888a9fdcb15e)
This is what I see on the front site. 
And this is what I expected to see on the front site.
![screen_846](https://github.com/rejetto/hfs/assets/172913982/15e5c52a-39f1-4404-a2e5-b444c290cf0a)

So the question is now:  Have I do something ?  
I have to less knowledge in style .css  to take action. So some help would be welcome

With regards,
Harry

**Comment by rejetto** [comment]
to see that, you got to disable tiles mode in "options".
it's not the theme

**Reply by Harry-nl** [reply]
Oke but I see not this option in version 52.9 under options also not in the options on the frontpage

**Comment by rejetto** [comment]
i meant options in the frontpage.
you don't have it?
screenshot please

**Reply by Harry-nl** [reply]
![screen_847](https://github.com/rejetto/hfs/assets/172913982/c63de563-f4c7-4465-8769-1c5a541634a4)

this is the frontpage section.  on the rest of the page there is no option for tiling

**Reply by Harry-nl** [reply]
![screen_848](https://github.com/rejetto/hfs/assets/172913982/0004942b-9e09-4a53-9de7-93aa81f909cd)

This the top section

**Comment by rejetto** [comment]
that's the admin-panel.
it's options in the front-end, user-end, the file explorer, whatever you want to call it.
feel free to suggest a name :)

<img width="335" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/6f179d5c-a245-4dea-bab7-a9986a7b7525">

**Reply by Harry-nl** [reply]
![screen_849](https://github.com/rejetto/hfs/assets/172913982/641a198c-1604-4ef9-8eeb-c0b8b01911b7)

you mean this one?

**Reply by rejetto** [reply]
yeah

**Comment by rejetto** [comment]
to say it all, in the admin-panel you have the corresponding option for that, it's "default tiles size", but as the name says, that's just the default, and you must have changed it at some point

**Reply by Harry-nl** [reply]
Ah I got it.... It was the tile setting  now set to 0 and working....  
Never to old to learn something...

Thanks rejetto>
by I love this this server

**Reply by rejetto** [reply]
brilliant!
